### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by maayanagranat**
Added Metrics Stream Engine data
Still need to link the transition plan here
and @AlonMagrafta to review

### CODE REVIEW

File: content/operate/rs/clusters/monitoring/_index.md [hunk 1]
```diff
@@ -13,14 +13,21 @@ weight: 96
 ---
 You can use the metrics that measure the performance of your Redis Enterprise Software clusters, nodes, databases, and shards
 to monitor the performance of your databases.
-In the Redis Enterprise Cluster Manager UI, you can see real-time metrics and configure alerts that send notifications based on alert parameters. You can also access metrics and configure alerts through the REST API.
+In the Redis Enterprise Cluster Manager UI, you can see metrics and configure alerts that send notifications based on alert parameters. You can also access metrics and configure alerts through the REST API.
```
Comment (mich-elle-luna) [inline content/operate/rs/clusters/monitoring/_index.md:16]:
```suggestion
In the Redis Enterprise Cluster Manager UI, you can view metrics, configure alerts, and send notifications based on alert parameters. You can also access metrics and configure alerts through the REST API.
```

File: content/operate/rs/clusters/monitoring/_index.md [hunk 2]
```diff
@@ -13,14 +13,21 @@ weight: 96
...
+
+Redis Enterprise vesion 7.8.2 introduces a new metrics stream engine that exposes new version of prometheus scraping endpoint https://<IP>:8070/v2  (In preview).
```
Comment (mich-elle-luna) [inline content/operate/rs/clusters/monitoring/_index.md:18]:
```suggestion
Redis Enterprise version 7.8.2 introduces a preview of the new metrics stream engine that exposes the the v2  Prometheus scraping endpoint at https://<IP>:8070/v2 .
```

File: content/operate/rs/clusters/monitoring/_index.md [hunk 3]
```diff
@@ -13,14 +13,21 @@ weight: 96
...
+This new engine exports all time-series metrics to external monitoring tools such as Grafana, DataDog, NewRelic and Dynatrace using Prometheus.
```
Comment (mich-elle-luna) [inline content/operate/rs/clusters/monitoring/_index.md:19]:
```suggestion
This new engine exports all time-series metrics to external monitoring tools such as Grafana, DataDog, NewRelic, and Dynatrace using Prometheus.
```

File: content/operate/rs/clusters/monitoring/_index.md [hunk 4]
```diff
@@ -13,14 +13,21 @@ weight: 96
...
+
+The new engine enables real-time monitoring, including full monitoring during maintenance operations, providing full visibility into performance during events such as shards' failovers and scaling operations.
+
+Users who are already using the existing scraping endpoint for integration, can follow this guide [link] to transition and try the new engine. It is possible to scrape both existing and new endpoints simultaneously, allowing in advanced dashboards preparation and smooth transition when ready.
```
Comment (mich-elle-luna) [inline content/operate/rs/clusters/monitoring/_index.md:23]:
```suggestion
If you are already using the existing scraping endpoint for integration, follow this guide [link] to transition and try the new engine. It is possible to scrape both existing and new endpoints simultaneously, allowing advanced dashboard preparation and a smooth transition.
```

File: content/operate/rs/clusters/monitoring/_index.md [hunk 5]
```diff
@@ -13,14 +13,21 @@ weight: 96
...
 
 To integrate Redis Enterprise metrics into your monitoring environment, see the integration guides for [Prometheus and Grafana]({{< relref "/integrate/prometheus-with-redis-enterprise/" >}}) or [Uptrace]({{< relref "/integrate/uptrace-with-redis-enterprise/" >}}).
 
 Make sure you read the [definition of each metric]({{< relref "/operate/rs/references/metrics/" >}})
 so that you understand exactly what it represents.
 
-## Real-time metrics
+## Cluster Manager Metrics
```
Comment (rrelledge) [inline content/operate/rs/clusters/monitoring/_index.md:30]: Small style suggestion
```suggestion
## Cluster manager metrics
```

**Comment by rrelledge** [comment]
@maayanagranat @AlonMagrafta It looks like CLAassistant wants you both to [sign the CLA agreement again](https://github.com/redis/docs/pull/852#issuecomment-2459522161).

**Review by rrelledge — APPROVED** [review]

**Pull request merged**